tests/integration/IVADefects/M1taxonomy/steps/returnAllDefectsStep.ts

```diff
@@ -0,0 +1,51 @@
+import {Given, When, Then} from "@cucumber/cucumber"
+import chai from "chai"
+import { generateToken } from "./common"
+import * as expectedJson from "../data/m1taxomomy.json";
+const {expect}=chai;
+var token : string;
+
+var baseURL="https://api.integration.cvs.dvsacloud.uk/integration/v3/"
+
+Given('A token is generated for authorization', async function () {
```

Comment:
I personally would prefer it like the following:
```suggestion
Given('A token is generated for authorization', async () => {
```
same for the rest of them.